packages/core/standard/src/errors.ts

```diff
@@ -1,2 +0,0 @@
-// TODO: add error codes
-export {};
```

Comment:
@jordansexton I created a separate `@wallet-standard/errors` package. Let me know if you prefer for the errors to live here instead.

Reply:
I think separate package is great

Reply:
Hữu jelmy